_posts/2024-11-11-this-month-in-servo.md

```diff
@@ -515,11 +515,14 @@ We’ve also landed optimisations in table layout (@Loirooriol, #33575) and in o
 
 Work continues on making our massive `script` crate build faster, with **improved incremental builds** (@sagudev, @mrobinson, #33502) and further patches towards **splitting `script` into smaller crates** (@sagudev, @jdm, #33627, #33665).
 
-## servoshell and embedding
+## servoshell, embedding, and devtools
+
+**[Devtools](https://book.servo.org/hacking/using-devtools.html) support** (`--devtools 6080`) is now compatible with Firefox 131+ (@eerii, #33661), and no longer lists iframes as if they were inspectable tabs (@eerii, #34032).
 
 [Servo-the-browser](https://book.servo.org/running-servoshell.html) now **avoids unnecessary redraws** (@webbeef, #34008), massively reducing its CPU usage, and no longer scrolls too slowly on HiDPI systems (@nicoburns, #34063).
 
-We’ve added a **new `--screen-size` option** that can help with testing mobile websites (@mrobinson, #34038), renaming the old `--resolution` option to `--window-size`, and we’ve **removed `--no-minibrowser` mode** (@Taym95, #33677).
+We’ve added a **new `--unminify-css` option**, allowing you to dump the CSS used by a page and modify it for debugging, [much like you can for JavaScript](https://book.servo.org/hacking/web-compat-bugs.html#diagnosing-js-errors).
```

Comment:
Only the dumping support is present. We haven't implemented support for modifying it yet.

Reply:
thanks! fixed in 4a1b1428f1ebe6ab8b8af9962e1fb65a7f132c4c